Doc/library/unittest.mock.rst

```diff
@@ -420,7 +420,7 @@ the *new_callable* argument to :func:`patch`.
         parameter as ``True``. Child mocks and the return value mock
         (if any) are reset as well.
 
-        .. note:: *return_value*, and :attr:`side_effect` are keyword-only
+        .. note:: :attr:`return_value`, and :attr:`side_effect` are keyword-only
```

Comment:
Nice! Attention to detail, increases consistency.

Reply:
@encukou , LMK if I need to look for anything else.

Reply:
Both of these refer to the function arguments, so they should be marked up as such (`*return_value*` and `*side_effect*`).

But, a bit above in: 

     In case you want to reset *return_value* or :attr:`side_effect`

these refer to attributes of the Mock objects, and should use `:attr:` (`return_value` should be changed to use `:attr:`).